src/process.js

```diff
@@ -1,2 +1,28 @@
 // Copied from https://github.com/hovancik/stretchly/blob/master/app/process.js - I think we should
 // update our app to have architecture similar to that.
+
+fetch("././dashboard/index.html")
```

Comment:
`./` needed twice?

Reply:
Update to `fetch("./dashboard/index.html")`